contracts/token/MerkleMine.sol

```diff
@@ -0,0 +1,182 @@
+pragma solidity ^0.4.17;
```

Comment:
We use `pragma solidity ^0.4.21` in https://github.com/livepeer/merkle-mine/blob/master/contracts/MerkleMine.sol but the current version of Truffle in this repo is older so switched this pragma statement to work with the version of Truffle used in this repo